elytron/src/test/resources/org/wildfly/extension/elytron/elytron-transformers-1.2-reject.xml

```diff
@@ -1,45 +1,5 @@
 <subsystem xmlns="urn:wildfly:elytron:3.0">
```

Comment:
shouldn't the XML namespace be `urn:wildfly:elytron:1.2`?

Reply:
No as I already said above - the version in this name is the version we are transforming to.